### DISCUSSION TITLE
Missing Funds/Withdrawal from Binancce to Firefly

### DISCUSSION THREAD

**Discussion by mingoat00** [iotaledger/firefly#3580] [Help & Support] [answered]
Hi i was withdrawing funds from Binance and copied the address that i generated in firefly. I withdrew the funds and the transaction has been noted as completed but the funds have not shown up in my account. The transaction is noted as complete on explorer and Binance. no funds have appeared in my firefly account . 

When i log into my firefly account it notes that there is an error but does not tell me what? it says network operational but messages per second is zero and reference rate is also zero . 

I have logged into the walllet on another device and no funds have appeared there also, but it it is receiving messages and referenced rate of 100%. 

Can anyone please help.

**Comment by MiKeF05** [comment]
Hello, is there anything in the error log of Firefly? You can go to the menu on the upper left, then a Firefly and error log. Do you use a VPN or a special firewall?

**Comment by mingoat00** [comment]
There is no error message. I have reverted it back to firefly 1.5 and it now says 

- network disconnected along with 
- zero receiving messages and 
- referenced rate of 0%. (note my typo above should have said 0% not 100%.


It appears there is a connection issue firefly on my computer and could possibly be impacted by the firewall. Here within lies the tricky bit. I cannot open up the windows security application on my laptop. It just crashes and have no way to switch the firewall off to see if that solves the issue or is there a backend way to access this? 

What are my options. 
Create a strong hold export. Is this a local file or does it firefly need to be connected to the log on the tangle?
If its local I can save a copy and open on another computer?
Will it save the receiving address that it generated? and my funds will be deposited once i open the stronghold on another computer.

**Comment by mingoat00** [comment, marked as answer]
update - i have resolved via using the more advanced balance finder for the funds on the other laptop via firefly . Pretty simple really 

Thanks

**Reply by Lekiah1** [reply]
Hello i have the same problem  but no error on my account and nothing new with advanced setting... how did you do ?

**Reply by mingoat00** [reply]
sorry for the late reply - in advanced setting - there was a balance finder that was able to locate the funds and confirmed the transaction into my account

**Reply by rafael-roeder** [reply]
I have the same problem, but my balance finder "Guthabenfinder" doesn`t find the balance. 
In my case there`s a error message appears. The Log says. 
"Wed, 17 Nov 2021 14:22:41 GMT
ClientError: {}"
But what does this mean?

**Reply by rafael-roeder** [reply]
The balance finder still can`t find my balance and delivers an error wish is not documented in the error history :(
The documentation of the transaction with my TxID at https://explorer.iota.org/mainnet/message/... Shows exactly the correct parameters with my transaction but I can not see it in my Firefly Wallet and the ammount is missing.

What con I Do,e.g. reset Firefly or do something else?
